cnxepub/collation.py

```diff
@@ -33,7 +33,8 @@ def easybake(ruleset, in_html, out_html):
     with respective read and write ability.
 
     """
-    html = etree.parse(in_html)
+    html_parser = etree.HTMLParser(encoding="utf-8")
```

Comment:
And that's where the bug shows up - the sorting. (Actually, now that I think about it, I'm sort of surprised _any_ of the selectors match …)